src/main/java/sirius/kernel/event/EventHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.event;
+
+/**
+ * Interface of EventHandler handling an event
```

Comment:
"Handles events dispatched by the {@link EventBus}" <- immer in 3. person beschreiben
< p >
Implementing classes needs to be registered using @Link Registered  in order...